README.md

```diff
@@ -1,37 +1,147 @@
+</a>
+
+<br />
+
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.6878318.svg)](https://doi.org/10.5281/zenodo.6878318)
+
+This repository contains a description of the scoring system used in the Monsoon Fantasy game and a simulation that was created to compare different scoring methods before the final scoring method was decided upon. The final scoring method takes into account both the risk and accuracy of a players guess. First, a potential maximum points value is determined for a guess. This value is higher the further a guess is from the historical rainfall average. Then, the player gets a percentage of their potential maximum points value depending on how close their guess is to the actual rainfall. Visit our [public GitHub repository](https://github.com/uaenvironment/monsoon-game-scoring-method) for more detailed information.
+
+![Monsoon Game Scoring Graph](https://github.com/uaenvironment/uaenvironment.github.io/blob/master/images/monsoon_game_scoring.png?raw=true)
+
+<br />
+
+### **Monsoon Post Game Analysis**
+When signing up to play Monsoon Fantasy, players had the option to fill out profile questions. These questions asked things such as
+- How many monsoon seasons have you experienced while living in the southwest?
+- How would you rate your understanding of the monsoon system?
+- During monsoon season how often do you consult different types of weather forecasts?
+
+All of the questions were optional and the responses were made anonymous before analysis. This data combined with the users' forecasts and points earned were used in this analysis which will be performed and the end of every monsoon season.
+
+![Monsoon Game Post Analysis](https://github.com/uaenvironment/uaenvironment.github.io/blob/master/images/monsoon_game_post.png?raw=true)
+
+<br />
+
+### **Monsoon Scraper**
+This project centralizes public data from several different flood control district (FCD) networks across the state of Arizona. This data is stored in a cloud based data warehouse and serves as the central data source for a number of monsoon related projects and research. To gather this data we have written a number of applications that run at different intervals dependent on the different FCD network implementations. These applications run on 15 minute to 1 hour intervals. These time intervals are required in order to obtain incremental precipitation data readings which are not available if  gathering data on an hour or day interval. In addition to precipitation data, some FCD sensors also report temperature, pressure, humidity, and stream flow intensity in washes.
+
+The result of this work can be found in a Frontiers in Climate publication titled [Curating and Visualizing Dense Networks of Monsoon Precipitation Data: Integrating Computer Science Into Forward Looking Climate Services Development](https://www.frontiersin.org/articles/10.3389/fclim.2021.602573/full).
+
+<br />
+
```

Comment:
```suggestion
### **Monsoon API**
The Monsoon API provides programmatic access to the data ingested by the Monsoon Scraper through an HTTP REST API interface. This enables other applications, such as the *Monsoon Plotter*, to utilize the dataset. 
```
I'm not sure what else to say here, but we should probably include that there is another application in here that's the target of the monsoon api package.

Reply:
No, There's no mention of the actual Monsoon API, the code that lives in the `monsoon-backend-api` repo.

Reply:
Added!